### PR TITLE
Feat/algo tokens

### PR DESCRIPTION
**Pull request by pancham88**
This PR adds support for `algo` in `account-lib` and has updates to support algo tokens.

The development has been done as part of ALGO Tokens initiative:
[ALGO Tokens JIRA Epic](https://bitgoinc.atlassian.net/browse/STLX-3652)

### CODE REVIEW

**Review by CapnMigraine — COMMENTED** [review]
@pancham88 [this commit](https://github.com/BitGo/BitGoJS/pull/1454/commits/49f793ab7bd6796fcc139f5cea1d7ec42a3914ae) was reviewed and merged into `feat/algo-tokens` before, but now it's missing. Can you cherry-pick it?

**Review by rliberal-simtlix — DISMISSED** [review]
LGTM.

**Review by shihao-guo — DISMISSED** [review]

**Review by shihao-guo — APPROVED** [review]

File: modules/account-lib/src/coin/algo/ifaces.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import algosdk from 'algosdk';
```
Comment (CapnMigraine) [inline modules/account-lib/src/coin/algo/ifaces.ts:1]: in some coins we call this file `iface.ts`, we should pick one and make it standard

**Review by CapnMigraine — APPROVED** [review]
lgtm

**Review by shihao-guo — APPROVED** [review]

**Pull request merged**